cmd/ingester/app/consumer/offset/manager.go

```diff
@@ -54,7 +54,7 @@ func NewManager(minOffset int64, markOffset MarkOffset, partition int32, factory
 	return &Manager{
 		markOffsetFunction:  markOffset,
 		close:               make(chan struct{}),
-		offsetCommitCount:   factory.Counter("offset-commit", map[string]string{"partition": strconv.Itoa(int(partition))}),
+		offsetCommitCount:   factory.Counter("offset-commit-count", map[string]string{"partition": strconv.Itoa(int(partition))}),
 		lastCommittedOffset: factory.Gauge("offset-commit", map[string]string{"partition": strconv.Itoa(int(partition))}),
```

Comment:
Counter-proposal:
* counter: offset-commits-total ("-total" is recommended by Prometheus docs)
* gauge: last-committed-offset

@vprithvi ^^^